HttpRecorder.py

```diff
@@ -0,0 +1,43 @@
+from Measurement import Measurement
+from Recorders import Recorder
+import json
+import urllib.request
+import sys
+
+def dict_format(template, measure: Measurement):
+    d = {}
+    # iterate over items in the dict
+    for key, item in template.items():
+        # if the item is a string, apply format
+        if type(item) == str:
+            formatted_value = item.format(
+                device_id=measure.device_id,
+                celsius=measure.get_celsius(),
+                fahrenheit=measure.get_fahrenheit(),
+                timestamp=measure.timestamp)
+            d[key] = formatted_value
+    return d
+
+
+class HttpRecorder(Recorder):
+    def __init__(self, config):
+        Recorder.__init__(self, 'http')
+        print("http recorder init")
+        self.template = config['json_template']
+        self.method = config['method']
```

Comment:
`method` is not used and since request is fixed to **HTTP POST** verb, it doesn't make sense to include in config.json or to be part of _HttpRecorder_ self properties.